backend/game/cards/discard_cards/victory_point_manipulation_cards/card_collector.py

```diff
@@ -0,0 +1,36 @@
+from os.path import dirname, basename, isfile, join
```

Comment:
NIT: When using a package like os, I prefer to just import the entire thing.  I know that there is nothing wrong with importing the os.path but I personally like to see the entire package in the name space especially because more often than not, a lot of functions are used in it (and just better readability for me)